src/subcommands/function/logs/mod.ts

```diff
@@ -0,0 +1,15 @@
+import { Command } from "cliffy";
+import { LogsOptions } from "./option.ts";
+import errorHandler from "@/error-handler.ts";
+
+import action from "./action.ts";
+
+const command = new Command<LogsOptions>()
+  .description("Print deployment logs")
```

Comment:
```suggestion
  .description("Print function logs")
```

一个 deployment 在后端是某个 env 下面的 function，这里 env 已经在另一个参数上体现了，到这里我感觉就可以直接用 function 了，并且这个 command 也是在 function 下面